tests/unit/tracerTest.cpp

```diff
@@ -0,0 +1,63 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+
+#include <tilck/common/basic_defs.h>
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+#include "mocking.h"
+
+using namespace testing;
+
+extern "C" {
+#include <tilck/common/string_util.h>
+#include <tilck/common/printk.h>
+
+#include <tilck/kernel/user.h>
+#include <tilck/mods/tracing.h>
+
+bool save_param_buffer(void *, long, char *, size_t);
```

Comment:
Add those declarations in `include/tilck/kernel/test/tracing.h` and include that header here. That header is (or should be) also included in the actual tracer's code. The purpose of the headers in kernel/test is precisely to avoid a signature mismatch error.